app/controllers/laa_references_controller.rb

```diff
@@ -22,12 +22,12 @@ def create
     if @link_attempt.valid?
       if link_laa_reference
         redirect_after_link
+        return
```

Comment:
use the "classic" method [shown here for now](https://blog.arkency.com/2014/07/4-ways-to-early-return-from-a-rails-controller/)

```
redirect_after_link and return
```